pyproject.toml

```diff
@@ -6,6 +6,7 @@ exclude = '''
   | \.tox
   | venv
   | build
+  | gen
```

Comment:
Note I made this change. This makes black's behavior consistent with flake8 & isort. otherwise it does reformat the generated protobuf files